myst_nbs/generalized_linear_models/GLM-simpsons-paradox.myst.md

```diff
@@ -524,7 +535,20 @@ The third and final model added a layer to the hierarchy, which captures our kno
 
 If you are interested in learning more, there are a number of other [PyMC examples](http://docs.pymc.io/nb_examples/index.html) covering hierarchical modelling and regression topics.
```

Comment:
this link needs to be updated. I guess the idea is to point to the homepage? if so you can use `` {doc}`index` ``